src/busco/version.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cat <<-END_VERSIONS
+busco: "$(busco --version 2>&1 | sed -n 's/BUSCO \([0-9.]*\)/\1/p')"
+END_VERSIONS
```

Comment:
For now, store the software versions directly in the docker container at `/var/software_versions.txt` -- [Example](https://github.com/viash-hub/biobase/blob/main/src/arriba/config.vsh.yaml#L383-L385).

Eventually this will be moved into a separate script, probably.